blog.html

```diff
@@ -0,0 +1,529 @@
+              >
+                Read More
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="popular-post">
+        <aside class="popular-posts">
+          <!-- Popular Posts section -->
+          <h4 class="text-l font-semibold mb-4">Popular Posts</h4>
+          <ul>
+            <li class="flex items-start py-4">
+              <img
+                src="assets\images\arthur-poulin-NhU0nUR7920-unsplash.jpg"
+                alt="Popular Post 1"
+                class="w-12 h-12 rounded mr-4"
+              />
+              <div>
+                <a href="/blog.html">
+                  <span class="text-12px font-semibold"
+                    >Traveling with Kids: Essential Items to Keep Your Kids
+                    Happy</span
+                  ></a
+                >
+              </div>
+            </li>
+            <li class="flex items-start py-4">
+              <img
+                src="assets\images\marissa-grootes-D4jRahaUaIc-unsplash 1.png"
+                alt="Popular Post 2"
+                class="w-12 rounded h-12 mr-4"
+              />
+              <div>
+                <a href="/blog.html">
+                  <span class="text-12px font-semibold"
+                    >Stay Organized on the Go: Best Travel Accessories</span
+                  ></a
+                >
+              </div>
+              <div style="margin-bottom: 50px"></div>
+            </li>
+            <li class="flex items-start py-4">
+              <img
+                src="assets\images\felix-rostig-UmV2wr-Vbq8-unsplash.jpg"
+                alt="Popular Post 3"
+                class="w-12 h-12 rounded mr-4"
+              />
+              <div>
+                <a href="/blog.html">
+                  <span class="text-12px font-semibold"
+                    >Top 10 Travel Accessories for Your Next Adventure</span
+                  ></a
+                >
+              </div>
+            </li>
+            <li class="flex items-start py-4">
+              <img
+                src="assets\images\tiffany-nutt-0ClfreiNppM-unsplash.jpg"
+                alt="Popular Post 4"
+                class="w-12 h-12 rounded mr-4"
+              />
+              <div>
+                <a href="/blog.html">
+                  <span class="text-12px font-semibold"
+                    >Eco-Friendly Travel: Accessories for Responsible
+                    Explorers</span
+                  ></a
+                >
+              </div>
+            </li>
+          </ul>
+        </aside>
+      </div>
+    </div>
+    <div class="button-container">
+      <button
+        type="button"
+        class="px-4 py-2 text-white text-base font-medium rounded-lg bg-travel-orange hover:bg-orange-600"
+      >
+        Load More Posts
+      </button>
+    </div>
+
+    <!-- Scroll Button -->
+    <button
+      onclick="topFunction()"
+      id="my-btn"
```

Comment:
Check this against your script file, the ID isnt the same, needs to be as per this line